### ISSUE TITLE
Only selected wheel should be edited when using TalkBack on Android

### ISSUE THREAD

**Issue by draperunner** [henninghall/react-native-date-picker#336]
**Describe the bug**
If you have activated TalkBack on Android, you are able to select the different wheels of the date picker. But if you use two fingers to change the value, you still need to touch the correct wheel. This can be very hard on the smallest wheels, like the one for hour. You might think you have selected the hour wheel, but when you scroll, the date changes because it is bigger and easier to touch.

**Expected behavior**
If TalkBack is active, only the selected wheel should be turned when swiping to edit.

I'm tagging @JencirJamal here since he made the great PR for accessibility support. :blush: https://github.com/henninghall/react-native-date-picker/pull/314

**Smartphone (please complete the following information):**
 - OS: Android
 - React Native version 0.63.4
 - react-native-date-picker version 3.3.0

**Comment by henninghall** [comment]
Release a fix for this in v3.4.3
